src/doc/trpl/crates-and-modules.md

```diff
@@ -460,21 +418,19 @@ Could not compile `phrases`.
 ```
 
 If we're importing multiple names from the same module, we don't have to type it out
-twice. Rust has a shortcut syntax for writing this:
+twice. Instead of this:
 
 ```{rust,ignore}
 use phrases::english::greetings;
 use phrases::english::farewells;
 ```
 
-You use curly braces:
+We can use this shortcut:
 
 ```{rust,ignore}
 use phrases::english::{greetings, farewells};
 ```
 
-These two declarations are equivalent, but the second is a lot less typing.
```

Comment:
It's also mentioned earlier:

> If we're importing multiple names from the same module, we don't have to type it out
twice. Rust has a shortcut syntax for writing this: